### ISSUE TITLE
Combobox: Wrong placement of aria-multiselectable attribute? ARIA attribute is not allowed: aria-multiselectable="true"

### ISSUE THREAD

**Issue by RIP21** [tailwindlabs/headlessui#2270]
**What package within Headless UI are you using?**

@headlessui/react

**What version of that package are you using?**

1.7.10

**What browser are you using?**
Chrome

**Reproduction URL**

Screenshot:
<img width="1197" alt="CleanShot 2023-02-10 at 14 36 54@2x" src="https://user-images.githubusercontent.com/3940079/218105515-615e8b74-d5db-4882-9405-49f780fe703d.png">

The multi-select implementation I have, for reference, is here: 
[Codesandbox](https://codesandbox.io/s/headless-bugs-ci0fb2?file=/src/index.tsx)

**Describe your issue**

[As per MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-multiselectable#example) `aria-multiselectable="true"` should be set on `ul` element (e.g. the container of the options not on the `input`). a11y checks (a11y-addon) of the Storybook complains about it.

I'm, by no means, an expert in ARIA attributes, but just noted this violation and I think it makes sense to report it.
